source/release_notes/4.0.27-22.rst

```diff
@@ -0,0 +1,19 @@
+.. _PSMDB-4.0.27-22:
+
+================================================================================
+*Percona Server for MongoDB* 4.0.27-22
+================================================================================
+
+:Date: September 28, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.0/install/index.html>`_
+
+Percona Server for MongoDB 4.0.27-22 is an enhanced, open source, and highly-scalable database that is a
```

Comment:
```suggestion
Percona Server for MongoDB 4.0.27-22 is an enhanced, source available, and highly-scalable database that is a
```

Reply:
Is there a way for us to prevent this from creeping in again and again? ;)

Reply:
Yes, there is a ticket to automate this https://jira.percona.com/projects/JIRABOT/issues/JIRABOT-104